packages/create-react-app/createReactApp.js

```diff
@@ -50,9 +50,39 @@ const tmp = require('tmp');
 const unpack = require('tar-pack').unpack;
 const url = require('url');
 const validateProjectName = require('validate-npm-package-name');
+const updateNotifier = require('update-notifier');
 
 const packageJson = require('./package.json');
 
+// Checks on every run, excluding the first.
+const { update } = updateNotifier({
+  pkg: packageJson,
+  updateCheckInterval: 0,
+});
+
+if (update) {
+  console.log();
+  console.error(
+    chalk.yellow(
+      `You are running \`create-react-app\` ${update.current}, which is behind the latest release (${update.latest}).\n\n` +
+        'We no longer support global installation of Create React App.'
+    )
+  );
+  console.log();
```

Comment:
whats this one for? An extra new line?

Reply:
It's convention we use a lot in the CRA scripts - I don't love it, but I think it's always good to follow convention.